Add the Common Desktop Environment

seal331 opened the pull request:
This PR adds the Common Desktop Environment, a DE commonly associated with commercial UNIX workstations of the 90s.

TODO list:
- proper systemd handling for CDE's dtlogin
- make a changelog entry
- comment out CDE category in index.xml until 12.3 releases

Zeckmathederg committed:
general.ent: Mention make-ca in *-use-internet.

seal331 committed:
cde: new category

seal331 committed:
cde: add libutempter

seal331 committed:
cde: add printproto

seal331 committed:
cde: add libXp

seal331 committed:
cde: add byacc

seal331 committed:
cde: add Motif

seal331 committed:
cde: add ksh

seal331 committed:
cde: add ncompress

seal331 committed:
cde: add CDE

seal331 committed:
cde/printproto: fix TODO

seal331 committed:
cde/*: address maintainer review

seal331 committed:
cde/libutempter: document

seal331 committed:
cde/libxp: document

seal331 committed:
cde/byacc: document

seal331 committed:
cde/byacc: fix index mishap

seal331 committed:
cde/motif: disable static; document

seal331 committed:
cde/ksh: document

seal331 committed:
cde/ncompress: document

seal331 committed:
cde/cde-de: disable static; document

Zeckmathederg reviewed (approved):
Looks good so far! I know it's still a draft, and those TODO points still need to be worked on, but things look good right now.

Zeckmathederg committed:
MIT-0 license: Removed.

Pretty sure we can just reuse the MIT license from GLFS and we are still
the copyright holders of that license, with the same dates. The MIT-0
License also just clutters up the book and can potentially make things
confusing. If this is a problem, please make a comment on this comment
or make an issue on https://github.com/glfs-book/lfs-qol/issues.

seal331 committed:
cde/libxp: fix undeclared dependency

seal331 committed:
cde/ksh: remove broken command

seal331 committed:
cde/motif: various bugfixes

seal331 committed:
cde/cde-de: implement systemd handling

seal331 committed:
organization: add CDE chapter

seal331 committed:
changelog: add stub CDE entry

seal331 committed:
svr4-tools/s-nail: fix index mishap

seal331 committed:
com-emu/idle: simplify ROM install commands

The sha512sums of both vidstate.rom's are the exact same.
Don't force people to go the extra mile to replace the
ROM with the exact same one from a different folder.

seal331 committed:
com-emu/{freebee,xhomer}: simplify install commands

seal331 committed:
changelog: fix merge conflict caused by #61

seal331 committed:
Merge branch 'trunk' into cde

seal331 commented:
The reason for 863ab781f80026f8c9e7d9a0130eda465d69b007 is that GitHub still thought I had a conflict despite the fact that I literally just fixed it from Git CLI. Ugh...

Zeckmathederg committed:
Merge branch 'trunk' into cde
